.travis.yml

```diff
@@ -1,3 +1,3 @@
 language: node_js
 node_js:
-  - 8
+  - 12
```

Comment:
Since we have no way to simulate true AppScript runtime, I think maybe we should just use "node" for the latest version?

Reply:
`8` doesn't work with the current version of Jest. Based on the travis docs, I think we actually have to list a number here, but am I mistaken?

Reply:
Instead of `- 12`, if you write `- node` it will always use the latest stable version of nodejs. So that we don't need to keep bumping the version.
https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions

Reply:
🤦 now i see!